<ui version="4.0" >
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>577</width>
    <height>412</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget" >
   <layout class="QGridLayout" name="gridLayout_3" >
    <item row="0" column="0" >
     <widget class="QTabWidget" name="tabWidget" >
      <property name="sizePolicy" >
       <sizepolicy vsizetype="Expanding" hsizetype="Expanding" >
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="currentIndex" >
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab_3" >
       <attribute name="title" >
        <string>Geometria</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_4" >
        <item row="0" column="0" >
         <widget class="QTableWidget" name="tableWidget_3" >
          <property name="sizePolicy" >
           <sizepolicy vsizetype="Expanding" hsizetype="Expanding" >
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="editTriggers" >
           <set>QAbstractItemView::NoEditTriggers</set>
          </property>
          <property name="tabKeyNavigation" >
           <bool>false</bool>
          </property>
          <property name="alternatingRowColors" >
           <bool>false</bool>
          </property>
          <property name="selectionMode" >
           <enum>QAbstractItemView::NoSelection</enum>
          </property>
          <property name="showGrid" >
           <bool>true</bool>
          </property>
          <row>
           <property name="text" >
            <string>Ancho de Modelo</string>
           </property>
          </row>
          <row>
           <property name="text" >
            <string>Altura de Modelo</string>
           </property>
          </row>
          <row>
           <property name="text" >
            <string>Cord. X Modelo</string>
           </property>
          </row>
          <row>
           <property name="text" >
            <string>Cord. Y Modelo</string>
           </property>
          </row>
          <row>
           <property name="text" >
            <string>Ancho</string>
           </property>
          </row>
          <row>
           <property name="text" >
            <string>Altura</string>
           </property>
          </row>
          <row>
           <property name="text" >
            <string>Cord. X</string>
           </property>
          </row>
          <row>
           <property name="text" >
            <string>Cord. Y</string>
           </property>
          </row>
          <column>
           <property name="text" >
            <string>Valor</string>
           </property>
          </column>
          <item row="0" column="0" >
           <property name="text" >
            <string>-</string>
           </property>
          </item>
          <item row="1" column="0" >
           <property name="text" >
            <string>-</string>
           </property>
          </item>
          <item row="2" column="0" >
           <property name="text" >
            <string>-</string>
           </property>
          </item>
          <item row="3" column="0" >
           <property name="text" >
            <string>-</string>
           </property>
          </item>
          <item row="4" column="0" >
           <property name="text" >
            <string>-</string>
           </property>
          </item>
          <item row="5" column="0" >
           <property name="text" >
            <string>-</string>
           </property>
          </item>
          <item row="6" column="0" >
           <property name="text" >
            <string>-</string>
           </property>
          </item>
          <item row="7" column="0" >
           <property name="text" >
            <string>-</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2" >
       <attribute name="title" >
        <string>Modelo</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout" >
        <item row="0" column="0" >
         <widget class="QTableWidget" name="tableWidget_2" >
          <property name="editTriggers" >
           <set>QAbstractItemView::NoEditTriggers</set>
          </property>
          <property name="tabKeyNavigation" >
           <bool>false</bool>
          </property>
          <property name="alternatingRowColors" >
           <bool>false</bool>
          </property>
          <property name="selectionMode" >
           <enum>QAbstractItemView::NoSelection</enum>
          </property>
          <row>
           <property name="text" >
            <string>Cobertura en Distancia</string>
           </property>
          </row>
          <row>
           <property name="text" >
            <string>Cobertura en Profundidad</string>
           </property>
          </row>
          <row>
           <property name="text" >
            <string>Color</string>
           </property>
          </row>
          <row>
           <property name="text" >
            <string>Leyenda</string>
           </property>
          </row>
          <row>
           <property name="text" >
            <string>Titulo</string>
           </property>
          </row>
          <row>
           <property name="text" >
            <string>Modelo</string>
           </property>
          </row>
          <column>
           <property name="text" >
            <string>Valor</string>
           </property>
          </column>
          <item row="0" column="0" >
           <property name="text" >
            <string>-</string>
           </property>
          </item>
          <item row="1" column="0" >
           <property name="text" >
            <string>-</string>
           </property>
          </item>
          <item row="2" column="0" >
           <property name="text" >
            <string>-</string>
           </property>
          </item>
          <item row="3" column="0" >
           <property name="text" >
            <string>-</string>
           </property>
          </item>
          <item row="4" column="0" >
           <property name="text" >
            <string>-</string>
           </property>
          </item>
          <item row="5" column="0" >
           <property name="text" >
            <string>-</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab" >
       <attribute name="title" >
        <string>Simulacion</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2" >
        <item row="0" column="0" >
         <widget class="QTableWidget" name="tableWidget" >
          <property name="enabled" >
           <bool>true</bool>
          </property>
          <property name="editTriggers" >
           <set>QAbstractItemView::NoEditTriggers</set>
          </property>
          <property name="alternatingRowColors" >
           <bool>false</bool>
          </property>
          <property name="selectionMode" >
           <enum>QAbstractItemView::NoSelection</enum>
          </property>
          <row>
           <property name="text" >
            <string>Distancia</string>
           </property>
          </row>
          <row>
           <property name="text" >
            <string>Profundidad</string>
           </property>
          </row>
          <row>
           <property name="text" >
            <string>Frecuencia Pico</string>
           </property>
          </row>
          <row>
           <property name="text" >
            <string>Frecuencia Maxima</string>
           </property>
          </row>
          <row>
           <property name="text" >
            <string>Titulo de Ventana</string>
           </property>
          </row>
          <row>
           <property name="text" >
            <string>Titula</string>
           </property>
          </row>
          <row>
           <property name="text" >
            <string>Tiempo (segs)</string>
           </property>
          </row>
          <row>
           <property name="text" >
            <string>Factor de Intensidad</string>
           </property>
          </row>
          <column>
           <property name="text" >
            <string>Valor</string>
           </property>
          </column>
          <item row="0" column="0" >
           <property name="text" >
            <string>-</string>
           </property>
          </item>
          <item row="1" column="0" >
           <property name="text" >
            <string>-</string>
           </property>
          </item>
          <item row="2" column="0" >
           <property name="text" >
            <string>-</string>
           </property>
          </item>
          <item row="3" column="0" >
           <property name="text" >
            <string>-</string>
           </property>
          </item>
          <item row="4" column="0" >
           <property name="text" >
            <string>-</string>
           </property>
          </item>
          <item row="5" column="0" >
           <property name="text" >
            <string>-</string>
           </property>
          </item>
          <item row="6" column="0" >
           <property name="text" >
            <string>-</string>
           </property>
          </item>
          <item row="7" column="0" >
           <property name="text" >
            <string>-</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>0</y>
     <width>577</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuPreferences" >
    <property name="title" >
     <string>&amp;Preferencias</string>
    </property>
    <addaction name="actionSizeSettings" />
   </widget>
   <widget class="QMenu" name="menuFile" >
    <property name="title" >
     <string>&amp;Archivo</string>
    </property>
    <addaction name="actionNew" />
    <addaction name="actionOpen" />
    <addaction name="actionSave" />
    <addaction name="actionSaveAs" />
    <addaction name="actionExit" />
   </widget>
   <widget class="QMenu" name="menuVelocityModel" >
    <property name="title" >
     <string>&amp;Modelo de Velocidad</string>
    </property>
    <addaction name="actionLoadModel" />
    <addaction name="actionModelParams" />
    <addaction name="actionPreview" />
   </widget>
   <widget class="QMenu" name="menuHelp" >
    <property name="title" >
     <string>A&amp;yuda</string>
    </property>
    <addaction name="actionAbout" />
   </widget>
   <widget class="QMenu" name="menuSimulation" >
    <property name="title" >
     <string>&amp;Simulacion</string>
    </property>
    <addaction name="actionSimParams" />
    <addaction name="actionRun" />
    <addaction name="actionViewTraces" />
   </widget>
   <addaction name="menuFile" />
   <addaction name="menuPreferences" />
   <addaction name="menuVelocityModel" />
   <addaction name="menuSimulation" />
   <addaction name="menuHelp" />
  </widget>
  <widget class="QStatusBar" name="statusbar" />
  <action name="actionSizeSettings" >
   <property name="text" >
    <string>&amp;Tamaño de Ventanas</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+G</string>
   </property>
  </action>
  <action name="actionExit" >
   <property name="text" >
    <string>&amp;Salir</string>
   </property>
  </action>
  <action name="actionModelParams" >
   <property name="text" >
    <string>&amp;Parametros del Modelo</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+M</string>
   </property>
  </action>
  <action name="actionPreview" >
   <property name="text" >
    <string>&amp;Vista Previa</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="actionEditModel" >
   <property name="text" >
    <string>&amp;Editar Modelo</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+E</string>
   </property>
  </action>
  <action name="actionSimParams" >
   <property name="text" >
    <string>&amp;Parametros de Simulacion</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+I</string>
   </property>
  </action>
  <action name="actionRun" >
   <property name="text" >
    <string>&amp;Ver Simulacion</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="actionNew" >
   <property name="text" >
    <string>Nuevo</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionSaveAs" >
   <property name="text" >
    <string>G&amp;uardar como</string>
   </property>
  </action>
  <action name="actionSave" >
   <property name="text" >
    <string>Guardar</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionAbout" >
   <property name="text" >
    <string>&amp;Sobre Viper</string>
   </property>
  </action>
  <action name="actionOpen" >
   <property name="text" >
    <string>&amp;Abrir</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+A</string>
   </property>
  </action>
  <action name="actionCreateModel" >
   <property name="text" >
    <string>Crear Modelo</string>
   </property>
  </action>
  <action name="actionLoadModel" >
   <property name="text" >
    <string>Cargar Modelo</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionViewTraces" >
   <property name="text" >
    <string>Ver &amp;Registro</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+L</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>actionExit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel" >
     <x>168</x>
     <y>171</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
